### ISSUE TITLE
Relative urls

### ISSUE THREAD

**Issue by Mateiadrielrafael** [google/typedoc-neo-theme#4]
Curently urls seem to be relative, so if i'm viewing a page in the `enums` directory and then click on a link from the navigator I end up like this: 
![image](https://user-images.githubusercontent.com/39400800/66865268-9038f080-ef9f-11e9-8341-43b7b2ea98f3.png)

This is my outline config: 
```json
 "outline": [
            {
                "Bits": {
                    "@reix/bits": "bits",
                    "@reix/rxjs-bits": "rxjs_bits",
                    "Usage with enums": "usageWithEnums",
                    "Usage with unions": "usageWithUnions",
                    "When to use this package": "whenToUseBits"
                },
                "Ecs": {
                    "@reix/ecs": "ecs"
                },
                "Dirty": {
                    "@reix/dirty": "dirty"
                }
            }
        ]
```

**Comment by Mateiadrielrafael** [comment]
it looks like just adding:
```html
<base href="/">```
in the head element fixed it.

**Comment by Mateiadrielrafael** [comment]
Nevermind, that only works locally but breaks github pages

**Comment by Fleker** [comment]
The AoG lib didn't have any enums, so we never ran into the problem. Seems like it comes from a lack of checking for enums in the NavigationAccordion: https://github.com/google/typedoc-neo-theme/blob/master/third_party/src/default/assets/js/src/typedoc/components/NavigationAccordion.ts#L83

**Comment by Mateiadrielrafael** [comment]
@Fleker thanks, when you have some time can you fix it and release 1.0.3?
